snapcraft-assets/snap/command-chain/alsa-launch

```diff
@@ -89,7 +98,10 @@ setup_alsa_override() {
 }
 
 if [ -f "$SNAP_USER_DATA/.snapcraft-alsa" ]; then
-  . "$SNAP_USER_DATA/.snapcraft-alsa"
+  mv "$SNAP_USER_DATA/.snapcraft-alsa" "$SNAP_USER_DATA/Always use PulseAudio"
```

Comment:
I'm wary of changing the filename here, and below, because people are already using this part and will be re-prompted even if they've previously told the script to always use pulseaudio...

Reply:
oh, I see, you're renaming the old file.. ignore that then :-)

Reply:
indeed -- that's exactly why it does that :-)